prow/tide/tide.go

```diff
@@ -246,8 +253,15 @@ func toSimpleState(s kube.ProwJobState) simpleState {
 
 // isPassingTests returns whether or not all contexts set on the PR except for
 // the tide pool context are passing.
-func isPassingTests(pr PullRequest) bool {
-	for _, ctx := range pr.Commits.Nodes[0].Commit.Status.Contexts {
+func isPassingTests(log *logrus.Entry, ghc githubClient, pr PullRequest) bool {
+	log = log.WithFields(pr.logFields())
+	contexts, err := headContexts(log, ghc, &pr)
+	if err != nil {
+		log.WithError(err).Error("Getting head commit status contexts.")
```

Comment:
Should this include repo & PR number?

Reply:
It does, a few lines above I add fields from the PR to the `logrus.Entry`. This includes the org, repo, number, and SHA.